CalcualtorLogic.js

```diff
@@ -13,11 +13,11 @@ class CalculatorLogic {
     };
     divide(number1, number2) {
         let result = number1 / number2;
+        if (number2 === 0) this.currentResult.textContent = 'Cannot divide by zero';
```

Comment:
can you explain how `this.currentResult.textContent` is can be used within this class

Reply:
with some way to connect this class with the other
same as what we did to connect the  Calculator class with class CalculatorLogic

Reply:
I have told you before that `this` refers to an object.
so which object `this` is referring to in your code.


you said "same as what we did to connect the Calculator class with class CalculatorLogic"
so what you did here?
![image](https://user-images.githubusercontent.com/16171515/112983045-1a095900-9155-11eb-94bc-bf9abf07bc11.png)

Reply:
Please do not push code and expecting it to magically work

Reply:
you did not answer me 
`this.currentResult.textContent`
the this keyword refers to which object?

Reply:
I did not read anywhere 

please !!

how we could send some thing to result 
?

1) doese we have a result ?
you told somthing which is more near to we do not have it 
2) if we have 
:
how we could change it without :
.textContent ?
 thanks